src/components/shared/Entity/ValidationErrorSummary.tsx

```diff
@@ -1,29 +1,47 @@
 import { Alert, AlertTitle, Collapse } from '@mui/material';
-import { KeyValue } from 'components/shared/KeyValueList';
+import KeyValueList, { KeyValue } from 'components/shared/KeyValueList';
 import { useRouteStore } from 'hooks/useRouteStore';
-import { map, uniq } from 'lodash';
-import { FormattedMessage } from 'react-intl';
+import { map } from 'lodash';
+import { FormattedMessage, useIntl } from 'react-intl';
 import { createStoreSelectors } from 'stores/Create';
 import { getStore } from 'stores/Repo';
 
 // TODO : Delete this or make is useful. Right now JSON Forms is kinda hard to write custom validation
 // just so that we don't have to display validation errors right away.
 function ValidationErrorSummary() {
+    const intl = useIntl();
+
     const entityCreateStore = getStore(useRouteStore());
     const [detailErrors, specErrors] = entityCreateStore(
         createStoreSelectors.errors
     );
     const displayValidation = entityCreateStore(
         createStoreSelectors.formState.displayValidation
     );
-    const filteredErrors = uniq(
-        map(detailErrors.concat(specErrors), 'instancePath')
-    );
 
-    const filteredErrorsList: KeyValue[] = filteredErrors.map((formError) => ({
-        title: formError,
-        val: formError,
-    }));
+    const filteredDetailErrors = map(detailErrors, 'instancePath');
+    const filteredSpecErrors = map(specErrors, 'instancePath');
+    // const filteredErrors = uniq(
+    //     map(filteredDetailErrors.concat(filteredSpecErrors), 'instancePath')
+    // );
```

Comment:
Commented instead of removed so that it is there as an example and hoping to add some of this related functionality back in soon. Would like to filter the errors for their paths and give the user a list of fields that have issues.